docs/user-guide/common-operations.md

```diff
@@ -1,9 +1,40 @@
-# Common Operations
+# Local Common Operations
 
-### Restarting the backend
+When running `./kubehound`, it will execute the 3 following action:
+
+* run the `backend` (graphdb, datadb and UI)
+* `dump` the kubernetes resources needed to build the graph
+* `ingest` the dumped data and generate the attack path for the targeted Kubernetes cluster.
+
+All those 3 steps can be run separatly.
+
+[![](../images/kubehound-local-commands.png)](../images/kubehound-local-commands.png)
+
+!!! Note
```

Comment:
see [docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)
```suggestion
> [!NOTE]
```

Reply:
nop, this is for the wiki, it is not the same. Run the `make local-wiki`